src/screens/NotificationsScreen.tsx

```diff
@@ -71,18 +70,22 @@ export const NotificationsScreen = () => {
       : noNotifications;
 
   return (
-    <View testID="notifications-screen">
-      <SafeAreaView>
-        <Divider />
+    <View testID="notifications-screen" style={styles.view}>
+      <ScrollView style={styles.scrollView}>
+        <Divider style={styles.dividerView} />
         {notificationsAreaContent}
-      </SafeAreaView>
+      </ScrollView>
     </View>
   );
 };
 
 const defaultStyles = createStyles('NotificationsScreen', () => ({
-  listItem: {},
-  icon: { paddingLeft: 16 },
+  view: {},
+  scrollView: {},
+  listItemView: {},
+  listItemContentsView: {},
+  dividerView: {},
+  iconView: { paddingLeft: 16 },
```

Comment:
Could this 16 be based on a theme spacing?  Or is better hard set to this number?